userguide/sound_classifier/export-coreml.md

```diff
@@ -111,6 +110,6 @@ for (label, sum) in prob_sums {
 }
 
 let most_probable_label = max_sum_label
-let probability = max_sum / Float(prob_sums.count)
+let probability = max_sum / Float(results.count)
```

Comment:
I don't think this is the right changes here. `prob_sums.count` is the number of categories. While `results.count` is the number of windows. Since we're summing across all windows, we want to divide by the number of windows (not the number of categories) in order to normalize the probabilities.

Reply:
So it's a right change. I use `results.count` instead of 'prob_sums.count'.